src/find-all-with-class.js

```diff
@@ -1,7 +1,20 @@
+import 'babel/polyfill';
```

Comment:
I don't think we can do this in a library as it will pollute the global library with the polyfill. We'll need to use the [`babel-runtime`](https://babeljs.io/docs/usage/runtime/)